src/content/en/showcase/2017/matterport2.md

```diff
@@ -0,0 +1,122 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-06-12 #}
+{# wf_updated_on: 2017-06-12 #}
+{# wf_featured_image: /web/showcase/2017/images/matterport/top.jpg #}
+{# wf_featured_snippet: At Matterport, we have a large library of over 450,000 3D Spaces that users can find embedded in sites all over the web. Users could even experience these Spaces in VR through a native mobile app for Cardboard or Gear VR. Now with WebVR support, it is much easier for users to experience VR right from their browser.<br/><br/>Launching WebVR support in our player required us to focus on the most essential features. This case study describes the process we followed for optimizing for performance. We're very happy with the final result that we achieved and we're excited about all the things we can do in the future with WebVR. #}
```

Comment:
The whole article is written by Matterport, with their byline. Note the start of the summary:
> At Matterport, we have...

Does this whole summary need redoing in Google voice?